<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmHistoricalTrends</class>
 <widget class="QDialog" name="frmHistoricalTrends">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1546</width>
    <height>1028</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QCustomPlot" name="widgetCustomplot" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widgetmenu" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="widget_select" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="labScreen">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>条件筛选</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gbox_time">
           <property name="title">
            <string>时间区间</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="1">
             <widget class="QDateTimeEdit" name="datetimeStart">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="displayFormat">
               <string>yyyy/M/d H:mm</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labStartTime">
              <property name="text">
               <string>开始时间</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labEndTime">
              <property name="text">
               <string>结束时间</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDateTimeEdit" name="dateTimeEnd"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gbox_speed">
           <property name="title">
            <string>转速筛选</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="4" column="0">
             <widget class="QLabel" name="labSpeedDown">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>下限</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" rowspan="3">
             <widget class="QLabel" name="labSpeedUp">
              <property name="text">
               <string>上限</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="cboxScreenSpeed">
              <property name="text">
               <string>启用</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" rowspan="2">
             <widget class="QSpinBox" name="sboxSpeedDown"/>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="sboxSpeedUp">
              <property name="suffix">
               <string/>
              </property>
              <property name="prefix">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gbox_valuesmlit">
           <property name="title">
            <string>幅值筛选</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QCheckBox" name="cboxScreenValue">
              <property name="text">
               <string>启用</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labValueUp">
              <property name="text">
               <string>上限</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="sboxValueUp">
              <property name="prefix">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labValueDown">
              <property name="text">
               <string>下限</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="sboxValueDown"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSelete">
           <property name="text">
            <string>设置筛选条件</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="widgetImageCheck">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labImageCheck">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>图像设置</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayoutImage">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item row="1" column="1">
            <widget class="QRadioButton" name="rbtn_variance">
             <property name="text">
              <string>方差</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QRadioButton" name="rbtn_peak_factor">
             <property name="text">
              <string>峰值因子</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QRadioButton" name="rbtn_margin_factor">
             <property name="text">
              <string>裕度因子</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QRadioButton" name="rbtn_peak_to_peak">
             <property name="text">
              <string>峰峰值</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="rbtn_min">
             <property name="text">
              <string>最小值</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="rbtn_mean">
             <property name="text">
              <string>均值</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QRadioButton" name="rbtn_rms">
             <property name="text">
              <string>均方根</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QRadioButton" name="rbtn_crest_factor">
             <property name="text">
              <string>波形因子</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="rbtn_max">
             <property name="text">
              <string>最大值</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QRadioButton" name="rbtn_rectifiedmean">
             <property name="text">
              <string>整流平均值</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QRadioButton" name="rbtn_kurtosis">
             <property name="text">
              <string>峭度</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QRadioButton" name="rbtn_Impulse_factor">
             <property name="text">
              <string>脉冲因子</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="rbtn_std_deviation">
             <property name="text">
              <string>标准差</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QRadioButton" name="rbtn_PointTem">
             <property name="text">
              <string>测点温度曲线</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QRadioButton" name="rbtn_AmbientTem">
             <property name="text">
              <string>环境温度曲线</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btnImageLoad">
           <property name="text">
            <string>图像加载</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
